### PR TITLE
Remove hard coded projectid

### PR DESCRIPTION
**Pull request by karlmutch**

### CODE REVIEW

**Comment by nieoh** [comment]
@karlmutch Does this mean we can keep a single README file? (README.rst)

File: studio/runner.py [hunk 1]
```diff
@@ -313,7 +313,7 @@ def submit_experiments(
 
             queue_name = 'pubsub_' + str(uuid.uuid4())
 
-            queue = PubsubQueue(queue_name, verbose=verbose)
+            queue = PubsubQueue(queue_name, config['database']['projectId'], verbose=verbose)
```
Comment (pzhokhov) [inline studio/runner.py]: now that I think about it - we should probably rather read projectId not from config, but from file to which GOOGLE_APPLICATION_CREDENTIALS points. Rationale is this - people might want to use central database (e.g. with project id studio-ed756), but their own queue. In fact, the way it is set up right now it won't work for anyone who is does not have admin rights or right credentials for studio-ed756 project

Comment (pzhokhov) [reply studio/runner.py]: fine with me for now

**Comment by karlmutch** [comment]
Tested using datacenter runner, pending future work in #140

**Pull request merged**